src/infra/core/src/auth/authentication_service_impl.rs

```diff
@@ -150,7 +148,7 @@ impl AuthenticationService for AuthenticationServiceImpl {
     fn supported_login_methods(&self) -> Vec<&'static str> {
         self.authentication_providers_by_method
             .keys()
-            .cloned()
+            .copied()
```

Comment:
That's a nice catch!

Reply:
Interesting trend, with the default clippy rule set, we got a change to `map(|v| v.clone())` -> `cloned()`.

Now, `cloned()` -> `copied()`. Just a note: migrations to more precise and rigorous practices -- like it!